influxdb3_py_api/src/system_py.rs

```diff
@@ -72,16 +72,27 @@ impl std::fmt::Debug for LogLine {
 
 #[pymethods]
 impl PyPluginCallApi {
-    fn info(&self, line: &str) -> PyResult<()> {
+    #[pyo3(signature = (*args))]
+    fn info(&self, args: &Bound<'_, PyTuple>) -> PyResult<()> {
+        let line = args
+            .try_iter()?
```

Comment:
Worth factoring out this little chain of calls since they are the same for all log levels?

Reply:
Ah, good point, updated in 81ec319f06d1bd5ca2b04fa4b11907dd633b6243